.github/workflows/ci.yml

```diff
@@ -9,20 +9,30 @@ on:
     branches-ignore: [ staging.tmp ]
 
 jobs:
-  build:
+
+  # Prepare a file containing the commit sha, to be included in the releases
+  prepare:
+    name: Prepare for builds
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Create commit-sha file
+      env:
+        GITHUB_SHA: ${{ github.sha }}
+      run: |
+          echo $GITHUB_SHA > commit-sha
+    - name: Publish commit-sha
+      uses: actions/upload-artifact@v2
+      with:
+        path: commit-sha
+        name: dist
+
+  # Builds to test that the package can be build on various platforms
+  test-build:
     name: ${{ matrix.name }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        name: [
-          MacOS Stable,
-          MacOS Nightly,
-          Ubuntu Stable,
-          Ubuntu Nightly,
-          Windows Stable,
-          Windows Nightly
-        ]
```

Comment:
Removed this, as it's not needed, I believe.

Reply:
Turns out this was here for bors, see `bors.toml` :)

Reply:
Does it though? The jobs still have the same names. I don't think Bors checks this file or anything, it just listens for success/fail events with matching names, so I think it's fine. Unless I'm missing something.